packages/components/forms/src/base-checkbox/BaseCheckbox.tsx

```diff
@@ -90,44 +90,47 @@ function _BaseCheckbox(
     typeof isChecked !== undefined ? isChecked : defaultChecked;
 
   return (
-    <Text
-      as="label"
-      fontColor="gray900"
-      className={cx(styles.wrapper, className)}
-      htmlFor={inputId}
-      testId={testId}
-      {...otherProps}
-    >
-      <input
-        {...inputProps}
-        aria-label={ariaLabel}
-        checked={isChecked}
-        defaultChecked={defaultChecked}
-        className={styles.input}
-        type={type === 'switch' ? 'checkbox' : type}
-        onChange={onChange}
-        onFocus={handleFocus}
-        onBlur={handleBlur}
-        onKeyDown={handleKeyDown}
-        value={value}
-        disabled={isDisabled}
-        role={type}
-        aria-checked={isIndeterminate ? 'mixed' : ariaChecked}
-        ref={inputRef}
-        required={isRequired}
-        aria-required={isRequired ? 'true' : undefined}
-        aria-invalid={isInvalid ? 'true' : undefined}
-        id={inputId}
-        name={name}
-      />
-      <GhostCheckbox
-        type={type}
-        isDisabled={isDisabled}
-        isIndeterminate={isIndeterminate}
-        size={size}
-      />
-      {children}
-    </Text>
+    <Box className={className}>
```

Comment:
I guess we can use `React.Fragment` here. Can we?

Reply:
Not with the way it's at the moment, because we only add the default spacing if the Checkbox/Radio is used within a group, and the context pass down the className with the styling and apply it here so it gets the same spacing in case there are checkboxes with helpText and some without helpText.
But it doesn't need to be a box necessarily either, or we could have the Context set a different prop that would be used to generate the styles on this component, so it knows if the spacing should be on the `helpText` or the `label`, or should we always add a bottom spacing regardless if it's inside a group or not??